internal/utils.go

```diff
@@ -284,30 +287,43 @@ func setTemplateOptions(tmpl map[string]interface{}, ignorePending bool, extraDe
 // by checking apiVersion and kind fields.
 // Return error when apiVersion and kind fields aren't string, or if the manifest
 // is a non-root policy manifest, but it is invalid because it is missing a name.
-func isPolicyTypeManifest(manifest map[string]interface{}) (bool, error) {
+func isPolicyTypeManifest(manifest map[string]interface{}) (bool, bool, error) {
```

Comment:
Could you please update the doc for the function now that there's a new return value?

Reply:
Good catch--done!